device/globals/action_menu.gd

```diff
@@ -16,6 +16,13 @@ func start(p_target):
 		target = p_target
 		target.connect("visibility_changed", self, "target_visibility_changed")
 
+		# Do not display the tooltip alongside the menu
+		if ProjectSettings.get_setting("escoria/ui/tooltip_follows_mouse"):
+			# FIXME: Hiding does not work :(
+			var hud = get_tree().get_root().get_node("/root/hud")
```

Comment:
`get_tree().get_root()` is equivalent to `get_node("/root")`

Isn't the path to the hud in the scene tree `get_node("/root/game/hud")` though?

Reply:
I tried `print_tree()` and got only the current tree, and I couldn't find it by trying, but maybe I didn't try that one.

What do you think of the addition to `ProjectSettings` for the hud? Is that valuable in restructuring the project?

Because if I can avoid exporting/exposing it, I'm more than happy to get rid of it. It doesn't feel right.

Reply:
A better solution would be to use `get_tree().call_group("hud", ...)` though:

http://docs.godotengine.org/en/3.0/classes/class_scenetree.html#class-scenetree-call-group

Reply:
I'll look into that after I get the mouse button stuff cleared, thanks!

It would be nice if this magically fixed the problem as well ;)

Reply:
I'm not sure it makes sense to make a singleton of hud, or at least it's not a quick fix, because there's still the hud that is instanced into the current scene and the singleton isn't a part of that scene, so I think there's a chance of things blowing up in unexpected ways.

Reply:
Either way, your `call_group()` solution was a good solution!

Squashed the two commits and force-pushing now.